src/test/java/org/junitpioneer/jupiter/RetryingTestExtensionTests.java

```diff
@@ -18,16 +18,24 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
 
+import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
 import org.junit.jupiter.api.TestTemplate;
+import org.junit.platform.testkit.engine.Execution;
 import org.junitpioneer.testkit.ExecutionResults;
 import org.junitpioneer.testkit.PioneerTestKit;
 import org.opentest4j.TestAbortedException;
 
 class RetryingTestExtensionTests {
 
+	private static final int SUSPEND_FOR = 200;
```

Comment:
Can you drop this to `10`? Should yield the same results but make the build half a second faster (well, we're multi-threading, so maybe not, but in theory!).